netconfdriver/service/operations/config_operations.py

```diff
@@ -30,14 +30,16 @@ def netconf_connect(host, port, user, password,rsa_key_path,kwargs):
     def netconf_edit(conn,package_properties,default_operation,request_id):
         try:
             external_request_id = str(uuid.uuid4())
-            ConfigOperations._generate_additional_logs(package_properties, 'sent', external_request_id,
-                                       'message', 'rpc', {'operation' : default_operation}, request_id)
+            ConfigOperations._generate_additional_logs(package_properties, 'sent', external_request_id, 'application/xml',
+                                       'request', 'netconf', {'default-operation' : default_operation}, request_id)
             edit_config_details = conn.edit_config(config=package_properties, target="running", default_operation=default_operation)
-            ConfigOperations._generate_additional_logs(edit_config_details, 'received', external_request_id,
-                                       'message', 'rpc', {'operation' : default_operation}, request_id)
+            ConfigOperations._generate_additional_logs(edit_config_details, 'received', external_request_id, 'application/xml',
+                                       'response', 'netconf', {'error' : '','errors' : edit_config_details.errors,'ok' : edit_config_details.ok }, request_id)
             logger.debug('config_details = %s', edit_config_details)
             return edit_config_details
         except Exception as e:
+            ConfigOperations._generate_additional_logs(e, 'received', external_request_id, 'plain/text',
+                                       'response', 'netconf', {'error-tag' : e.tag,'error-info' : '','error-severity' : e.severity,'error-path' : e.path,'error-type' : e.type}, request_id)
```

Comment:
I have checked all values,  except the e.info all properties print with respective values. So I changed it and put it as an empty string.

Here is the Response

```
{
 "@timestamp": "2023-03-24T10:29:44.120Z", 
 "@version": "1", 
 "message": "Request could not be completed because the relevant data model content already exists.", 
 "host": "netconf-driver-597f7949cc-dw69t", 
 "path": "/usr/local/lib/python3.10/site-packages/netconfdriver/service/operations/config_operations.py", 
 "tags": [],
 "type": "logstash", 
 "thread_name": "MainThread", 
 "level": "INFO",
 "logger_name": "netconfdriver.service.operations.config_operations",
 "tracectx.tenantid": "1", 
 "tracectx.processid": "e9d3c2c1-bb25-497d-a5e5-215bd70a0fd1",
 "tracectx.transactionid": "9ec49d1b-4478-4d51-a344-e6740d68695d",
 "tracectx.taskid": "7",
 "content_type": "plain/text",
 "message_direction": "received", 
 "tracectx.externalrequestid": "b9d861fe-09ae-49f9-bb60-023935f093bf",
 "message_type": "response", 
 "protocol": "netconf", 
 "protocol_metadata": "{\"error-tag\": \"data-exists\", \"error-info\": \"\", \"error-severity\": \"error\", \"error-path\": \"/mynetconf:netconflist/netconf[netconf-id=\"1253\"]\", \"error-type\": \"application\"}", 
 "tracectx.driverrequestid": "Create::e8073995-2d2d-42a2-b5cb-7b0a1cbb8613"
 }
```